src/types.ts

```diff
@@ -369,6 +369,22 @@ export interface Price {
     price: BigNumber;
 }
 
+interface BasePriceResponse {
+    price: BigNumber;
+    buyAmount: BigNumber;
+    sellAmount: BigNumber;
+    sellTokenAddress: string;
+    buyTokenAddress: string;
+    sources: GetSwapQuoteResponseLiquiditySource[];
+}
+
+export interface GetSwapQuotePriceResponse extends BasePriceResponse {
```

Comment:
Can we rename this to just `GetSwapPriceResponse`?  Since `SwapQuote` and `SwapPrice` are two distinct things, it feels a little weird to see `...SwapQuotePrice...`.